### PR TITLE
Updated 7260 64x100 MMU Profile.

### PR DESCRIPTION
**Pull request by abdosi**
What I did:

Updated 7260 MMU Profile based on latest `MSFT Tier 1 Tomahawk2_MMU_Setting_48x100G_40m_16x100G_300m_v1.0` and 
`TH2_PGHdrm_MSFT`.

How I verify:
Made sure image is up/traffic is flowing/mmu dump looked fine.
SAI qos test need will be updated to support this SKU.

### CODE REVIEW

**Review by yxieca — DISMISSED** [review]

File: device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/buffers_defaults_t0.j2 [hunk 1]
```diff
@@ -10,18 +10,22 @@
...
     "BUFFER_POOL": {
         "ingress_lossless_pool": {
-            "size": "33329088",
+            {# SS Tab2 Row 71 #}
+            "size": "33555392",
             "type": "ingress",
             "mode": "dynamic",
-            "xoff": "7827456"
+            {# SS Tab2 Row 68 #}
+            "xoff": "8992256"
```
Comment (neethajohn) [inline device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/buffers_defaults_t0.j2:18]: For T0, the headroom pool needs to be calculated based on 40m uplinks and 5m downlinks. also 8 uplinks and remaining as downlinks. @baiwei0427 , comments?

**Review by neethajohn — CHANGES_REQUESTED** [review]

File: device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/buffers_defaults_t0.j2 (hunk 1 above)
Comment (abdosi) [reply device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/buffers_defaults_t0.j2:18]: @neethajohn: can we update t0 later once we have usecase

Comment (baiwei0427) [reply device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/buffers_defaults_t0.j2:18]: @abdosi  What cable assumptions do you hold when calculating buffer values?

Comment (abdosi) [reply device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/buffers_defaults_t0.j2:18]: @baiwei0427 as i mention above change for t0 are same as t1 for now just to keep them synch. Idea of this PR is to update  for T1 only. I think for T0 we need to revisit once we have use-case

Comment (neethajohn) [reply device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/buffers_defaults_t0.j2:18]: @baiwei0427 , the assumptions for T1 are 16 uplinks 300m and 48 downlinks 40m.

**Review by neethajohn — DISMISSED** [review]
LGTM. please wait for @baiwei0427 to sign off as well

File: device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/buffers_defaults_t0.j2 (hunk 1 above)
Comment (abdosi) [reply device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/buffers_defaults_t0.j2:18]: yes @neethajohn

Comment (neethajohn) [reply device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/buffers_defaults_t0.j2:18]: @abdosi , can we remove the numbers for T0 and update it later with the correct values. the pg_profile_lookup.ini is not going to change for T0

Comment (abdosi) [reply device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/buffers_defaults_t0.j2]: @neethajohn updated to remove t0 buffer config and also reverted 100G 5m pg_profile value.

**Comment by abdosi** [comment]
@baiwei0427 can you please sign-off.

**Review by baiwei0427 — APPROVED** [review]

**Review by neethajohn — APPROVED** [review]

**Comment by prsunny** [comment]
@abdosi , @neethajohn , is this also required for 202012?

**Pull request merged**